sunpy/visualization/limb.py

```diff
@@ -1,99 +0,0 @@
-from matplotlib import patches
-
-import astropy.units as u
-from astropy.constants import R_sun
-from astropy.coordinates import SkyCoord
-
-from sunpy.coordinates.frames import HeliocentricInertial, Helioprojective
-from sunpy.coordinates.sun import _angular_radius
-from sunpy.coordinates.utils import get_limb_coordinates
-from sunpy.visualization import wcsaxes_compat
-from sunpy.visualization.draw import _plot_vertices
-
-__all__ = ['draw_limb']
```

Comment:
We can not delete this wholesale, we will need to go through a deprecation cycle. 
An example is https://docs.sunpy.org/en/latest/dev_guide/contents/public_api.html?highlight=deprecation#deprecation-policy-and-breaking-changes 

You will want to restore the file, either keep the existing function or import the old one if it has the same API and wrap it here.